src/Unosquare.RaspberryIO.Playground/System/System.Info.cs

```diff
@@ -1,46 +1,48 @@
-﻿namespace Unosquare.RaspberryIO.Playground
+namespace Unosquare.RaspberryIO.Playground
 {
     using Computer;
-    using Swan;
-    using Swan.Logging;
     using System;
     using System.Linq;
-    using System.Threading.Tasks;
 
     public static partial class SystemTests
     {
-        private static async Task TestSystemInfo()
+        private static void TestSystemInfo()
         {
             Console.Clear();
 
-            $"GPIO Controller initialized successfully with {Pi.Gpio.Count} pins".Info();
-            $"{Pi.Info}".Info();
+            Console.WriteLine($"GPIO Controller initialized successfully with {Pi.Gpio.Count} pins");
+            Console.WriteLine($"{Pi.Info}");
 
             try
             {
-                $"BoardModel {Pi.Info.BoardModel}".Info();
-                $"ProcessorModel {Pi.Info.ProcessorModel}".Info();
-                $"Manufacturer {Pi.Info.Manufacturer}".Info();
-                $"MemorySize {Pi.Info.MemorySize}".Info();
+                Console.WriteLine($"BoardModel {Pi.Info.BoardModel}");
+                Console.WriteLine($"ProcessorModel {Pi.Info.ProcessorModel}");
+                Console.WriteLine($"Manufacturer {Pi.Info.Manufacturer}");
+                Console.WriteLine($"MemorySize {Pi.Info.MemorySize}");
             }
-            catch
+            catch (Exception x)
             {
-                // ignore
+                Console.WriteLine("Error obtaining system configuration: " + x.Message);
             }
 
-            $"Uname {Pi.Info.OperatingSystem}".Info();
-            $"HostName {NetworkSettings.Instance.HostName}".Info();
-            $"Uptime (seconds) {Pi.Info.Uptime}".Info();
-            var timeSpan = Pi.Info.UptimeTimeSpan;
-            $"Uptime (timespan) {timeSpan.Days} days {timeSpan.Hours:00}:{timeSpan.Minutes:00}:{timeSpan.Seconds:00}"
-                .Info();
-
-            (await NetworkSettings.Instance.RetrieveAdapters().ConfigureAwait(false))
-                .Select(adapter => $"Adapter: {adapter.Name,6} | IPv4: {adapter.IPv4,16} | IPv6: {adapter.IPv6,28} | AP: {adapter.AccessPointName,16} | MAC: {adapter.MacAddress,18}")
-                .ToList()
-                .ForEach(x => x.Info());
-
-            Terminal.WriteLine(ExitMessage);
+            try
+            {
+                Console.WriteLine($"Uname {Pi.Info.OperatingSystem}");
+                Console.WriteLine($"HostName {NetworkSettings.Instance.HostName}");
+                Console.WriteLine($"Uptime (seconds) {Pi.Info.Uptime}");
+                var timeSpan = Pi.Info.UptimeTimeSpan;
+                Console.WriteLine($"Uptime (timespan) {timeSpan.Days} days {timeSpan.Hours:00}:{timeSpan.Minutes:00}:{timeSpan.Seconds:00}");
+
+                var adapters = NetworkSettings.Instance.RetrieveAdapters();
```

Comment:
I would prefer the use of `GetAwaiter()` if you want to remove the async/await. By the way any particular reason?

Reply:
Using Async/await has no benefit here, since we're anyway waiting for the whole info to be printed (i.e. there's no window that needs updating at the same time). So we can as well do it synchronously, which simplifies the code IMHO. 
According https://docs.microsoft.com/de-de/dotnet/api/system.runtime.compilerservices.configuredtaskawaitable.getawaiter?view=netframework-4.8#System_Runtime_CompilerServices_ConfiguredTaskAwaitable_GetAwaiter GetAwaiter() shouldn't be used in user code, so just calling the Result property is the preferred way of synchronously getting the result of a Task.

Reply:
Yeah, but the issue with Result is the AggregateException (https://stackoverflow.com/questions/17284517/is-task-result-the-same-as-getawaiter-getresult#targetText=To%20achieve%20that%2C%20you%20can,has%20a%20GetResult()%20method.).

For this particular case, no big deal.